src/TypeScript.UnitTests/EslintBridgeClient/EslintBridgeProcessTests.cs

```diff
@@ -57,6 +58,54 @@ public void Start_FailsToFindNode_FileNotFoundException()
             act.Should().ThrowExactly<FileNotFoundException>().And.Message.Should().Contain("node.exe");
         }
 
+        [TestMethod]
+        public async Task Start_SecondCall_ProcessFailed_StartsNewProcess()
+        {
+            var fakeNodeExePath = CreateScriptThatPrintsPortNumber(123, isHanging: true);
+            var startupScriptPath = "dummy path";
+
+            var nodeLocator = SetupNodeLocator(fakeNodeExePath);
+            var testSubject = CreateTestSubject(startupScriptPath, nodeLocator: nodeLocator.Object);
+
+            await testSubject.Start();
+
+            var oldProcess = testSubject.Process;
+
+            oldProcess.HasExited.Should().BeFalse();
+            oldProcess.Kill();
+            await oldProcess.WaitForExitAsync();
+            oldProcess.HasExited.Should().BeTrue();
+
+            await testSubject.Start();
+
+            var newProcess = testSubject.Process;
+            newProcess.Should().NotBeSameAs(oldProcess);
+            newProcess.HasExited.Should().BeFalse();
+            newProcess.Kill();
+        }
+
+        [TestMethod]
+        public async Task Start_SecondCall_ProcessIsStillRunning_DoesNotStartNewProcess()
+        {
+            var fakeNodeExePath = CreateScriptThatPrintsPortNumber(123, isHanging: true);
+            var startupScriptPath = "dummy path";
+
+            var nodeLocator = SetupNodeLocator(fakeNodeExePath);
+            var testSubject = CreateTestSubject(startupScriptPath, nodeLocator: nodeLocator.Object);
+
+            await testSubject.Start();
+
+            var oldProcess = testSubject.Process;
+            oldProcess.HasExited.Should().BeFalse();
+
+            await testSubject.Start();
+
+            var newProcess = testSubject.Process;
+            newProcess.Should().BeSameAs(oldProcess);
```

Comment:
Same cleanup problem: if the assertions fail the processes will be left running.